packages/static-wado-deploy/lib/studiesMain.mjs

```diff
@@ -1,10 +1,18 @@
 import commonMain from "./commonMain.mjs";
 import uploadDeploy from "./uploadDeploy.mjs";
 import uploadIndex from "./uploadIndex.mjs";
+import { retrieveF, retrieveMain } from "./retrieveDeploy.mjs";
 
 export default async function (studyUID, options) {
-  console.log("studyUID=", studyUID);
   const studyDirectory = studyUID ? `studies/${studyUID}` : "studies";
+  if( options.retrieve ) {
+    console.log("Retrieve studyUID", studyUID);
+    await retrieveMain(this, "root", options, retrieveF.bind(null, studyDirectory));
```

Comment:
I believe there is no need to bind retrieveF and even pass it here. We can simplify the function by:
```

retrieveDeployConfig(context, deployPropertyName, options,  directory) {

....

 const {deployPlugin, deployments = []} = config;
  console.log(`Retrieve ${name}`, deployPlugin, options);
  
  if(deployments.length) {
  .....
return   Promise.all(
....
retrieveConfigFor(directory,config, name, options, deployPlugin);
....

)
  .....
  }
  

}
```
Ex of call
`retrieveDeployConfig(this, "root", options,  studyDirectory)`